packages/gatsby-source-prismic-graphql/src/gatsby-node.ts

```diff
@@ -154,6 +156,7 @@ const getDocumentsQuery = ({
         edges {
           cursor
           node {
+            ${extraPageFields}
```

Comment:
Does an empty line (e.g. when the user doesn't set extraPageFields) cause any issues in the query?

Reply:
No, it's fine. Double checked this and also we should've seen the tests fail if it did cause issues.

Reply:
Ok, cool 👍